concepticondata/references/references.bib

```diff
@@ -2070,3 +2070,10 @@ @book{Reinhard1970
 year={1970},
 address ={Kathmandu}
 }
+
+@misc{Carling2017,
```

Comment:
There is an official publication in PLOS ONE on the project. In case they do not provide an official how-to-quote statement, I urge to use this one, as it represents DIACL. See [here](http://bibliography.lingpy.org/evobib.php?key=Carling2018) for the bibtex, etc.

Reply:
The PLOS ONE article focusses on the Typology side of DIACL, though. So in terms of a database description, [this PDF](https://diacl.ht.lu.se/Content/documents/DiACL-lexicology.pdf) (linked from the database landing page) is probably the best we can get.

Reply:
I'd still say that the misc is not enough to quote it, right? I'd say: quote it as a `@book`, and give the university as publisher, etc., but also include all people involved. Ah, just saw the PDF, this also gives the authors.